### ISSUE TITLE
Playground.jl build LoadError: ReadOnlyMemoryError()

### ISSUE THREAD

**Issue by erwinyusrizal** [rofinn/Playground.jl#83]
Hi I tried to add package Playground and the installation failed due to error, `LoadError: ReadOnlyMemoryError()` and I tried to build it again it returns the same error, any idea ?

Thanks

**Comment by rofinn** [comment]
Can you post full output of `versioninfo(true)` and `Pkg.add("Playground")`?

**Comment by erwinyusrizal** [comment]
@rofinn thanks for get back to me, below is the screenshot

![selection_989](https://user-images.githubusercontent.com/3752374/42362468-920ccbe4-811d-11e8-96ab-1f5e02c316ca.png)

![selection_990](https://user-images.githubusercontent.com/3752374/42362473-96244cb6-811d-11e8-8b6d-9254a6a1598c.png)

Thanks

**Comment by rofinn** [comment]
Hmmm, that's failing on `chmod(join(build_dir, "playground"), "+x")`. Maybe this is an issue with FilePaths.jl? Could you try creating a file in that directory and making it executable?

```
julia> using FilePaths

julia> test_file = join(Path(Pkg.dir("Playground")), p"deps/usr/build/Foo.txt")
p"//Users/rory/.julia/v0.6/Playground/deps/usr/build/Foo.txt"

julia> touch(test_file)
Base.Filesystem.File(false, RawFD(-1))

julia> mode(test_file)
-rw-r--r--

julia> chmod(test_file, "+x")

julia> mode(test_file)
-rwxr-xr-x

julia> remove(test_file)

```

If that still fails it might be worth double checking that `chmod` works on that file from the command line.

**Comment by erwinyusrizal** [comment]
Hi @rofinn I got the same error when running `mode(test_file)` eventho I already set chmod the build folder to be executable

![selection_996](https://user-images.githubusercontent.com/3752374/42404207-73131e16-81b1-11e8-8c35-dddba277b843.png)


![selection_995](https://user-images.githubusercontent.com/3752374/42404152-1968adfe-81b1-11e8-8398-60bec04d17d3.png)

Thank you,

**Comment by rofinn** [comment]
Yep, looks like this might be a libc compatibility issue :( Can you try running `ldd --version`?
